packages/flame/lib/src/test_helpers/expect_vector2.dart

```diff
@@ -2,10 +2,11 @@ import 'package:test/test.dart';
 
 import '../../extensions.dart';
 
+// Default is that one pixel away from the expected value is accepted
```

Comment:
this is a big change, is it to solve some existing flakyness or newly introduced one?

Reply:
It is for the existing one when the `SequenceEffect` is `isInfinite` and `isAlternating`.

Reply:
If this is for a small number of tests only, then wouldn't it be better to pass `epsilon=1` in those specific tests? That way all other tests can remain more strict.

Reply:
True, I'll do that